src/renderers/shared/fiber/ReactFiberTreeReflection.js

```diff
@@ -114,3 +114,14 @@ exports.findCurrentHostFiber = function(component : ReactComponent<any, any, any
   }
   return null;
 };
+
+exports.getComponentName = function(fiber: Fiber): string {
+  const type = fiber.type;
+  const instance = fiber.stateNode;
+  const constructor = instance && instance.constructor;
+  return (
+    type.displayName || (constructor && constructor.displayName) ||
+    type.name || (constructor && constructor.name) ||
+    'A Component'
+  );
+};
```

Comment:
We already have the `getComponentName` module that you could use:

https://github.com/facebook/react/blob/master/src/shared/utils/getComponentName.js#L18-L38